concepts/numbers/about.md

```diff
@@ -1,3 +1,107 @@
 # About
 
-TODO: add information on numbers concept
+There are two different kinds of numbers in JavaScript - numbers and bigints
+
+Numbers are the most used, and represent numeric data type in the double-precision 64-bit floating point format.
+
+- `number`: a numeric data type in the double-precision 64-bit floating point format (IEEE 754).
+  Examples are `-6`, `-2.4`, `0`, `0.1`, `1`, `3.14`, `16.984025`, `25`, `976`, `1024.0` and `500000`.
+- `bigint`: a numeric data type that can represent _integers_ in the arbitrary precision format.
+  Examples are `-12n`, `0n`, `4n`, and `9007199254740991n`.
+
+```javascript
+let numericValue = 42;
+// => 42
+```
+
+A number literal like `42` in JavaScript code is a floating-point value, not an integer.
+There is no separate integer type in common everyday use.
+The `BigInt` type was not designed to replace `Number` for everyday uses.
+`42` is still a `Number`, not a `BigInt`.
+
+Number may also be expressed in literal forms like `0b101`, `0o13`, `0x0A`. Learn more on numeric lexical grammar [here][lexical-grammar].
+
+## Built-in Object
+
+There are two built-in objects that are useful when dealing with numbers:
+
+- [`Number`][built-in-number]: static properties for common / useful values, static methods for [type-checking][type-checking] and [type-conversion][type-conversion], instance methods for [type-conversion][type-conversion] and [formatting numbers as strings][string-formatting].
+- [`Math`][built-in-math]: properties and methods for mathematical constants and functions, does **not** work with `BigInt`.
+
+The `Number` built-in object is _also_ a global `function` that can be used to convert _almost anything_ that can be represented as a `number` to a `number`. It is less forgiving then _parsing_ a string to a number.
```

Comment:
Not sure if the backticks are being used as quotes, here or if we are talking about actual code items.  Also "represented as a number to a number, I don't think this language is correct.  Things are represented as other things, but not "to" something.  I find this paragraph is somewhat unclear in terms of that portion.